frontend/src/components/explorer/mapViewer/MapsListing.vue

```diff
@@ -2,31 +2,33 @@
   <div id="maps-listing">
     <div v-for="category in Object.keys(mapsListing).filter(c => mapsListing[c].length > 0).sort()"
          :key="category" class="card my-3">
-      <p class="is-capitalized is-size-6 has-text-weight-bold">{{ category.replace(/.$/," maps") }}</p>
-      <span v-for="item in mapsListing[category]" :key="item.id">
-        <template v-if="showing2D">
-          <template v-if="item.svgs.length === 0">
-            {{ item.name }}
-          </template>
-          <template v-else-if="item.svgs.length === 1">
-            <a @click="changeToMap(item.svgs[0].id)">
-              {{ item.svgs[0].customName }}
-            </a>
+      <template v-if="category !== 'customs' || showing2D">
+        <p class="is-capitalized is-size-6 has-text-weight-bold">{{ category.replace(/.$/," maps") }}</p>
+        <span v-for="item in mapsListing[category]" :key="item.id">
+          <template v-if="showing2D">
+            <template v-if="item.svgs.length === 0">
+              {{ item.name }}
+            </template>
+            <template v-else-if="item.svgs.length === 1">
+              <a @click="changeToMap(item.svgs[0].id)">
+                {{ item.svgs[0].customName }}
+              </a>
+            </template>
+            <template v-else>
+              {{ item.name }}:
+              <a v-for="svg in [...item.svgs].sort((a, b) => a.customName.localeCompare(b.customName))"
+                :key="svg.id" class="inline" @click="changeToMap(svg.id)">
```

Comment:
The linter wants another space here but maybe this will get auto-resolved if you apply the fix mentioned in the other message since you would be able to remove the `<template v-if="showing2D">` wrapper.

Reply:
Hi Shan, it seems the logic `<template v-if="showing2D">` needs to be kept since the treatment for the svg maps are slightly different from the 3D maps. I've added an extra space to solve the linter warning message anyway.

Reply:
Ok, I think we are on the same page here. I meant that the logic for `showing2D` should still be kept and moved into the computed property and then the actual `<template>` can be removed, which looks like to be exactly what you have done.